server/src/helpers/game/commands/directions/directionalmovement.ts

```diff
@@ -0,0 +1,22 @@
+import { Direction, IMacroCommandArgs, IPlayer } from '../../../../interfaces';
+import { MacroCommand } from '../../../../models/macro';
+
+export class DirectionalMovement extends MacroCommand {
+  aliases = ['e', 'w', 's', 'n', 'nw', 'ne', 'sw', 'se'];
+  canBeFast = true;
+
+  execute(player: IPlayer, args: IMacroCommandArgs) {
+    const totalMoves = Array<Direction>();
+    totalMoves.push(args.calledAlias as Direction);
+
+    if (args.stringArgs) {
+      const additionalMoves = args.stringArgs.split(' ');
+      additionalMoves.forEach(element => totalMoves.push(element as Direction));
+    }
+
+    totalMoves.slice(0, player.stats.move || 3).forEach(element => {
```

Comment:
use `this.game.characterHelper.getStat(player, Stat.Move)` instead, it will give you their current move stat